matrix-api.c

```diff
@@ -32,16 +32,8 @@
 
 #include "libmatrix.h"
 #include "matrix-json.h"
-
-struct _MatrixApiRequestData {
```

Comment:
It is part of the api and it is needed so that the structs exist in other files that include the api header.